src/assets/metadata/ipfs_hashes.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Balancer": {
```

Comment:
Not sure the cowswap repo is the right place for storing this info.
Maybe a gist would make more sense, to de-couple from one UI?